app/templates/frameworks/contract_variation.html

```diff
@@ -69,6 +69,8 @@
           <div class="section-description">
           {% if variation_details.get('countersignedAt') and agreed_details %}
             {{ variation.variation_description_in_place }}
+          {% elif not variation_details.get('countersignedAt') and agreed_details and variation.variation_description_not_in_place_but_agreed %}
```

Comment:
The logic here is reasonably complex so it would be nice to have some additional tests around the expected text in different cases.

Reply:
@TheDoubleK added in new test:
https://github.com/alphagov/digitalmarketplace-supplier-frontend/pull/788/files#diff-860e899ae62cbab98fee8fa871bcc53bR4994